services/client/src/Components/DragDrop/DragDrop.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react'
+import './DragDrop.css';
+
+
+// tutorial pulled from:
+// https://medium.com/@650egor/simple-drag-and-drop-file-upload-in-react-2cb409d88929
+//
+
+class DragDrop extends Component {
+
+    dropRef = React.createRef()
+
+    state = {
+        dragging: false
+    }
+
+    handleDrop = (e) => {
+        e.preventDefault()
+        e.stopPropagation()
+        this.setState({drag: false})
+        if (e.dataTransfer.files && e.dataTransfer.files.length > 0) {
+            this.props.handleDrop(e.dataTransfer.files)
+            e.dataTransfer.clearData()
+            this.dragCounter = 0
+        }
+    }
+
+    handleDrag = (e) => {
```

Comment:
you should be able to do this with css instead of writing an explicit helper function that tells react to do nothing

Reply:
okay cool. Do you have an example of what that might look like?